text/text.go

```diff
@@ -300,3 +302,51 @@ func Draw(dst *ebiten.Image, text string, face font.Face, x, y int, clr color.Co
 
 	textM.Unlock()
 }
+
+// MeasureString measures the size of a given string using a given font.
+//
+// text is the string that's being measured.
+// face is the font for text rendering.
+//
+// Be careful that the passed font face is held by this package and is never released.
+// This is a known issue (#498).
```

Comment:
Just to confirm: this is because `glyphAdvance`, right?

Reply:
I couldn't tell you. I copied this comment from `Draw()` because the code is very similar, so it would make sense that the same bug would happen there too.